tool/sel4coreplat/sel4.py

```diff
@@ -819,22 +819,23 @@ def get_id(self, kernel_config: KernelConfig) -> int:
     # ARM Page Directory
     Sel4Label.ARMPageDirectoryMap: 42,
     Sel4Label.ARMPageDirectoryUnmap: 43,
+#    Sel4Label.ARMSMCCall: 44,
```

Comment:
should this be commented out?

Reply:
> should this be commented out?

No, it's a hint where the "missing" number went

Reply:
Ah right. I would leave it uncommented as all the invocation labels, some of which aren't used by seL4CP at all, are all uncommented right now.

Reply:
Done in https://github.com/Ivan-Velickovic/sel4cp/pull/7/commits/6a1c8ef31edbb8302f242876b1e77a611c608e17